schemas/2019-06-01/Microsoft.Storage.json

```diff
@@ -99,72 +68,96 @@
+                "$ref": "#/definitions/storageAccounts_encryptionScopes_childResource"
               },
               {
-                "$ref": "#/definitions/storageAccounts_encryptionScopes_childResource"
+                "$ref": "#/definitions/storageAccounts_blobServices_childResource"
               },
               {
-                "$ref": "#/definitions/storageAccounts_objectReplicationPolicies_childResource"
+                "$ref": "#/definitions/storageAccounts_fileServices_childResource"
               },
               {
-                "$ref": "#/definitions/storageAccounts_privateEndpointConnections_childResource"
+                "$ref": "#/definitions/storageAccounts_queueServices_childResource"
               },
               {
-                "$ref": "#/definitions/storageAccounts_managementPolicies_childResource"
+                "$ref": "#/definitions/storageAccounts_tableServices_childResource"
               }
             ]
           }
+        },
+        "sku": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/Sku"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression"
+            }
+          ],
+          "description": "The SKU of the storage account."
+        },
+        "tags": {
+          "oneOf": [
+            {
+              "type": "object",
+              "additionalProperties": {
+                "type": "string"
+              },
+              "properties": {}
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression"
+            }
+          ],
+          "description": "Gets or sets a list of key value pairs that describe the resource. These tags can be used for viewing and grouping this resource (across resource groups). A maximum of 15 tags can be provided for a resource. Each tag must have a key with a length no greater than 128 characters and a value with a length no greater than 256 characters."
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.Storage/storageAccounts"
+          ]
         }
       },
       "required": [
-        "name",
-        "type",
         "apiVersion",
-        "sku",
         "kind",
-        "location"
+        "location",
+        "name",
+        "sku",
+        "type"
       ],
       "description": "Microsoft.Storage/storageAccounts"
     },
     "storageAccounts_blobServices": {
       "type": "object",
       "properties": {
+        "apiVersion": {
+          "type": "string",
+          "enum": [
+            "2019-06-01"
+          ]
+        },
         "name": {
           "oneOf": [
             {
               "type": "string",
-              "enum": [
-                "default"
-              ]
+              "pattern": "^.*/default$"
```

Comment:
Please make sure the name really works.

Reply:
The test cases have passed. See https://github.com/Azure/azure-resource-manager-schemas/pull/1130/files#diff-e90beca61fcb273a8c3d021d52534605